server/target/target.go

```diff
@@ -118,11 +163,16 @@ func readTargets(ctx context.Context, env environment.Env, req *trpb.GetTargetRe
 		q.AddWhereClause("i.branch_name = ?", branchName)
 	}
 	q.SetOrderBy("t.label ASC, i.created_at_usec", false /*=ascending*/)
+	return fetchTargetsFromDB(ctx, env, q, tq.GetRepoUrl())
+}
+
+func fetchTargetsFromDB(ctx context.Context, env environment.Env, q *query_builder.Query, repoURL string) (*trpb.GetTargetResponse, error) {
 	queryStr, args := q.Build()
 
 	seenTargets := make(map[string]struct{}, 0)
 	targets := make([]*trpb.Target, 0)
 	statuses := make(map[string][]*trpb.TargetStatus, 0)
+	var paginationToken *PaginationToken
```

Comment:
nit: consider calling it `nextPageToken` (I got confused thinking this was being used to paginate the query made within this func)